### PR TITLE
Paging over GetStreamsToSnapshot fails

### PR DESCRIPTION
**Pull request by andlju**
Wrote this on the forums: https://groups.google.com/forum/#!topic/neventstore/58wSRcmRWgQ

Managed to reproduce the error in an acceptance test. Fails both for MsSQL and Sqlite (don't know about the others)

/Anders

### CODE REVIEW

**Comment by andlju** [comment]
OK, so I added a StreamId parameter to the query, and that seems to have solved the problem and makes the paging work.

**Comment by andreabalducci** [comment]
thanks, I'll check as soon as possible

File: src/NEventStore/Persistence/Sql/SqlDialects/CommonSqlStatements.resx [hunk 1]
```diff
@@ -187,6 +187,7 @@ AND NOT EXISTS
     ON C.BucketId = @BucketId
    AND C.StreamId = S.StreamId
    AND C.StreamRevision &gt;= S.StreamRevision
+ WHERE C.StreamId &gt; COALESCE(@StreamId, '0')
```
Comment (andreabalducci) [inline src/NEventStore/Persistence/Sql/SqlDialects/CommonSqlStatements.resx]: StreamId is a string, why we should check if greater than zero as fallback?

Comment (andlju) [inline src/NEventStore/Persistence/Sql/SqlDialects/CommonSqlStatements.resx]: I honestly don't remember why I put that there..

**Comment by alkampfergit** [comment]
Closed due to inactivity.

**Pull request closed without merging**